examples/plot_second_level_button_press.py

```diff
@@ -58,21 +58,56 @@
 design_matrix = pd.DataFrame([1] * len(second_level_input),
                              columns=['intercept'])
 
-second_level_model = SecondLevelModel(smoothing_fwhm=8.0)
+second_level_model = SecondLevelModel(smoothing_fwhm=8.0, n_jobs=1)
 second_level_model = second_level_model.fit(second_level_input,
                                             design_matrix=design_matrix)
 
-##########################################################################
+#########################################################################
+# Estimate uncorrected effects
+# --------------------------------
 # To estimate the contrast is very simple. We can just provide the column
 # name of the design matrix.
 z_map = second_level_model.compute_contrast(output_type='z_score')
 
-###########################################################################
-# We threshold the second level contrast at uncorrected p < 0.001 and plot
+#########################################################################
+# We threshold the contrast at uncorrected p < 0.001 and plot
 p_val = 0.001
 z_th = norm.isf(p_val)
 display = plotting.plot_glass_brain(
-    z_map, threshold=z_th, colorbar=True, plot_abs=False, display_mode='z',
-    title='group left-right button press (unc p<0.001')
+    z_map, threshold=z_th, colorbar=True, plot_abs=False, display_mode='ortho',
+    title='group left-right press (unc p < 0.001)')
+
+plotting.show()
+
+#########################################################################
+# Estimate FWE corrected effects with permutation test
+# ----------------------------------------------------
+# We use a similar function that also allows to tune the permutation
+# parameters. For this example we will only use 1000 permutations. But we
+# recommend to use 10000. First we compute the two_sided_test.
+z_map = second_level_model.compute_contrast_permutations(
+    output_type='cor_z_score', n_perm=1000, two_sided_test=True)
```

Comment:
Sure, but isn't that confusing ? I'm really afraid of people making things wrong.